sdk-core-protos/build.rs

```diff
@@ -131,5 +136,58 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
             ],
         )?;
 
+    let file_path = out.join("temporal.api.common.v1.rs");
+    post_process_payload_file(file_path)?;
```

Comment:
So I just saw in the docs there's a built-in thing on the configurator: https://docs.rs/prost-build/0.13.1/prost_build/struct.Config.html#method.skip_debug

I think we might not need to do any manual replacement.

Reply:
Aaaah, I see. That would be awesome, yes. In the meantime looks like you could potentially use this https://docs.rs/tonic-build/latest/tonic_build/struct.Builder.html#method.compile_with_config to interact directly with the prost config

Reply:
https://github.com/hyperium/tonic/pull/1900 created, using `compile_with_config` in the meantime, thanks for the suggestion!